app/domain/authentication/authn_jwt/authentication_parameters.rb

```diff
@@ -0,0 +1,23 @@
+module Authentication
+  module AuthnJwt
+    # Data class to store data regarding jwt token that is needed during the jwt authentication process
+    class AuthenticationParameters
+      attr_accessor :jwt_identity, :decoded_token, :jwt_token
+      attr_reader :authenticator_name, :service_id, :account, :username, :client_ip, :request
+
+      def initialize(authentication_input:, jwt_token:)
+        @authenticator_name = authentication_input.authenticator_name
+        @service_id = authentication_input.service_id
+        @account = authentication_input.account
+        @username = authentication_input.username
+        @client_ip = authentication_input.client_ip
+        @request = authentication_input.request
+        @jwt_token = jwt_token
+      end
+
+      def authenticator_resource_id
```

Comment:
what does resource id mean? it looks like it returns the value of the `service_id` variable so shouldn't this method be named `service_id`?

Reply:
it is the authenticator resource id perfix for accessing the authenticator variables.

We have a technical dept ticket to change it

Reply:
Its resource ID becaouse it return id of variable in the db. 
Calling it `variable_id` can be better?

Reply:
how about `authn_jwt_variable_id`?

Reply:
Refactoring it to authn_jwt_variable_id
But this will be improved more to resource command class  in the technical depth after merge to master